extensions/apiconnect/README.md

```diff
@@ -3,7 +3,9 @@
 This module extends LoopBack with the ability to integrate with
 [IBM API Connect](https://www.ibm.com/cloud/api-connect). It comes with an
 `ApiConnectComponent` that adds an `OASEnhencer` extension to contribute
-`x-ibm-configuration` to the OpenAPI spec generated by LoopBack applications.
+`X-IBM-Configuration` to the OpenAPI spec generated by LoopBack applications. It
```

Comment:
just in case.. the `x-ibm-configuration` needs to be all lower case in order to work in API Connect.